elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -718,10 +720,15 @@ public PersistentResource getRelation(String relation, String id) {
     }
 
     private Set<PersistentResource> getRelation(String relationName, boolean checked) {
-        if (requestScope.getTransaction() != null && !requestScope.getPredicates().isEmpty()) {
-            final Class<?> entityClass = dictionary.getParameterizedType(obj, relationName);
-            final String valType = dictionary.getBinding(entityClass);
-            final Set<Predicate> filters = new HashSet<>(requestScope.getPredicatesOfType(valType));
+        boolean hasPredicates = !requestScope.getPredicates().isEmpty();
+        if (hasPredicates || !requestScope.getSorting().isDefaultInstance()
+                || requestScope.getPagination().isDefault()) {
```

Comment:
I don't understand this.  What does pagination default have to do with building filters?